packages/relay/src/lib/eth.ts

```diff
@@ -1017,8 +1017,17 @@ export class EthImpl implements Eth {
         }
       }
 
+      let cachedResponse = this.cache.get(`eth_call: ${call.data} from ${call.to}`);
```

Comment:
nit: add const for label. Maybe shorten key also
```suggestion
      const cacheKey = `eth_call:.${call.to}.${call.data}`;
      let cachedResponse = this.cache.get(cacheKey);
```